analysis/import/CSVImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/util/ParserDialogHelper.kt

```diff
@@ -0,0 +1,30 @@
+package de.maibornwolff.codecharta.importer.util
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import java.io.File
+import java.nio.file.Paths
+
+class ParserDialogHelper {
+
+    companion object {
+        fun getInputFiles(isSourceMonitor: Boolean): MutableList<String> {
```

Comment:
I just made this class to avoid some duplication, only thing new here is the KInquirer hint.